nautobot/core/checks.py

```diff
@@ -40,6 +40,13 @@
     obj=settings,
 )
 
+W006 = Warning(
```

Comment:
@glennmatthews I went with a warning because there is technically nothing stopping you from doing it, if you accept the risk of invalid results.

This is not a hill I will die on though, if we want to make this an error.